sphinx_gallery/scrapers.py

```diff
@@ -73,14 +91,17 @@ def matplotlib_scraper(block, block_vars, gallery_conf, **kwargs):
     """
     matplotlib, plt = _import_matplotlib()
     image_path_iterator = block_vars['image_path_iterator']
-    image_paths = list()
+    fig_titles = ''
```

Comment:
Shouldn't need this line, `fig_titles` is only used inside the loop and it's set on line 104